docs/database-engine/availability-groups/windows/replication-subscribers-and-always-on-availability-groups-sql-server.md

```diff
@@ -75,6 +75,7 @@ Go
 EXEC sp_addpullsubscription_agent 
         @publisher =  N'<publisher name>', 
         @subscriber = N'<availability group listener name>',
+        @Distributor = N'<Distributor availability group listener name>', -- this parameter should only be used if the Distribution database is part of an availability group. 
```

Comment:
```suggestion
        @distributor = N'<distributor availability group listener name>', -- this parameter should only be used if the distribution database is part of an availability group. 
```

Reply:
@rwestMSFT This looks good.

Reply:
Please approve the changes - I don't have the ability to do so.